From 55f857bad58b9a8f97e804c0103a0ed7e5a25536 Mon Sep 17 00:00:00 2001
From: Adrian <146006794+adrian4096@users.noreply.github.com>
Date: Tue, 28 Oct 2025 10:26:04 +0100
Subject: [PATCH 021/133] std.heap.debug_allocator outdated doc (#25634)
X-Developer-Signature: v=1; a=openpgp-sha256; l=907; i=xtex@astrafall.org;
 h=from:subject; bh=oGfk9JZ1zjP2CUnpoCF02l7kUeV8OuYyB0VoktmjdVU=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cn+/eV5iePnH3b+tS8JO6JpUpzR79DWJPGiTuhLCE
 r3vFvOHjoksDGJcDJZiiixFhg3erDrp/KLLymVh5rAygQyRFmlgYGBgYGHgy03MKzXSMdIz1TbU
 MzTUMdIxYuDiFICplqxg+MO9qipMUHFzVc0H/mdx5+ZVXucwnONu2mD6wH2hBM+38NuMDBPv3z0
 mdXuVrtMhi5WGj3o3zoov37ijPe3UbkNbrYKgB9wA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

Fixed a relatively small outdated doc string, referring to the bucket linked list.

(cherry picked from commit 4e9dd099c5d624b101222bfc4e6f415b5375f8d7)
---
 lib/std/heap/debug_allocator.zig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/std/heap/debug_allocator.zig b/lib/std/heap/debug_allocator.zig
index a4b1de5b47b5..a8b88c16e07f 100644
--- a/lib/std/heap/debug_allocator.zig
+++ b/lib/std/heap/debug_allocator.zig
@@ -69,7 +69,7 @@
 //! versa.
 //!
 //! When a bucket is full, a new one is allocated, containing a pointer to the
-//! previous one. This singly-linked list is iterated during leak detection.
+//! previous one. This doubly-linked list is iterated during leak detection.
 //!
 //! Resizing and remapping work the same on small allocations: if the size
 //! class would not change, then the operation succeeds, and the address is
-- 
2.52.0

